src/ir_Daikin.cpp

```diff
@@ -3,13 +3,14 @@ An Arduino sketch to emulate IR Daikin ARC433** remote control unit
 Read more at:
 http://harizanov.com/2012/02/control-daikin-air-conditioner-over-the-internet/
 
-Copyright 2016 sillyfrog
+Copyright 2017 sillyfrog
```

Comment:
Leave the earlier year. IIRC earlier time/year is better. Feel free to change it to "2016, 2017" if you want .. and it doesn't make the cpplint.py program complain. ;_